administrator/language/de-DE/plg_system_stats.ini

```
@@ -5,24 +5,24 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SYSTEM_STATS="System - Joomla!-Statistikerhebung"
-PLG_SYSTEM_STATS_BTN_NEVER_SEND="Niemals"
-PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Immer"
+PLG_SYSTEM_STATS_BTN_NEVER_SEND="Nein"
+PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Ja, Joomla! helfen"
```

```suggestion
PLG_SYSTEM_STATS_BTN_SEND_ALWAYS="Ja"
```

Wie im orginalen Issue finde ich es nicht gut wenn wir suggestive sprache benutzen. Dies wird ja u.a. auch absehbar bei den Cookie Dingern auf EU ebene verboten. Ich persönlich würde daher hiermit überhaupt nicht erst anfagen.